docs/src/guide/data_modeling.md

```diff
@@ -28,13 +33,79 @@ Notes:
 
```

Comment:
In the above section (Process), there are broken links that should go the Built-in Collabs page (my fault)